incubator/mssql/templates/pvc.yaml

```diff
@@ -0,0 +1,23 @@
+{{- if .Values.persistence.enabled }}
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    app: {{ template "fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+  annotations:
+  {{- if .Values.persistence.storageClass }}
+    volume.beta.kubernetes.io/storage-class: {{ .Values.persistence.storageClass | quote }}
+  {{- else }}
+    volume.alpha.kubernetes.io/storage-class: default
```

Comment:
Don't support the ancient alpha annotation any more.

Use this PR as reference for storage class support: https://github.com/kubernetes/charts/pull/1332/files

Here's an example for persistent volumes: https://github.com/kubernetes/charts/blob/master/stable/prometheus/templates/server-deployment.yaml#L87-L92